apps/condo/migrations/20241009105402-0433_payment_rawaddress_payment_recipient_and_more.js

```diff
@@ -0,0 +1,58 @@
+// auto generated by kmigrator
+// KMIGRATOR:0433_payment_rawaddress_payment_recipient_and_more:IyBHZW5lcmF0ZWQgYnkgRGphbmdvIDUuMSBvbiAyMDI0LTEwLTA5IDA1OjU0Cgpmcm9tIGRqYW5nby5kYiBpbXBvcnQgbWlncmF0aW9ucywgbW9kZWxzCgoKY2xhc3MgTWlncmF0aW9uKG1pZ3JhdGlvbnMuTWlncmF0aW9uKToKCiAgICBkZXBlbmRlbmNpZXMgPSBbCiAgICAgICAgKCdfZGphbmdvX3NjaGVtYScsICcwNDMyX2JhbmthY2NvdW50X25hbWVfYmFua2FjY291bnRoaXN0b3J5cmVjb3JkX25hbWUnKSwKICAgIF0KCiAgICBvcGVyYXRpb25zID0gWwogICAgICAgIG1pZ3JhdGlvbnMuQWRkRmllbGQoCiAgICAgICAgICAgIG1vZGVsX25hbWU9J3BheW1lbnQnLAogICAgICAgICAgICBuYW1lPSdyYXdBZGRyZXNzJywKICAgICAgICAgICAgZmllbGQ9bW9kZWxzLlRleHRGaWVsZChibGFuaz1UcnVlLCBudWxsPVRydWUpLAogICAgICAgICksCiAgICAgICAgbWlncmF0aW9ucy5BZGRGaWVsZCgKICAgICAgICAgICAgbW9kZWxfbmFtZT0ncGF5bWVudCcsCiAgICAgICAgICAgIG5hbWU9J3JlY2lwaWVudCcsCiAgICAgICAgICAgIGZpZWxkPW1vZGVscy5KU09ORmllbGQoYmxhbms9VHJ1ZSwgbnVsbD1UcnVlKSwKICAgICAgICApLAogICAgICAgIG1pZ3JhdGlvbnMuQWRkRmllbGQoCiAgICAgICAgICAgIG1vZGVsX25hbWU9J3BheW1lbnRoaXN0b3J5cmVjb3JkJywKICAgICAgICAgICAgbmFtZT0ncmF3QWRkcmVzcycsCiAgICAgICAgICAgIGZpZWxkPW1vZGVscy5UZXh0RmllbGQoYmxhbms9VHJ1ZSwgbnVsbD1UcnVlKSwKICAgICAgICApLAogICAgICAgIG1pZ3JhdGlvbnMuQWRkRmllbGQoCiAgICAgICAgICAgIG1vZGVsX25hbWU9J3BheW1lbnRoaXN0b3J5cmVjb3JkJywKICAgICAgICAgICAgbmFtZT0ncmVjaXBpZW50JywKICAgICAgICAgICAgZmllbGQ9bW9kZWxzLkpTT05GaWVsZChibGFuaz1UcnVlLCBudWxsPVRydWUpLAogICAgICAgICksCiAgICBdCg==
+
+exports.up = async (knex) => {
+    await knex.raw(`
+    BEGIN;
+--
+-- [CUSTOM] Set Statement Timeout to some large amount - 25 min (25 * 60 => 1500 sec)
+--
+SET statement_timeout = '1500s';
+--
+-- Add field rawAddress to payment
+--
+ALTER TABLE "Payment" ADD COLUMN "rawAddress" text NULL;
+--
+-- Add field recipient to payment
+--
+ALTER TABLE "Payment" ADD COLUMN "recipient" jsonb NULL;
+--
+-- Add field rawAddress to paymenthistoryrecord
+--
+ALTER TABLE "PaymentHistoryRecord" ADD COLUMN "rawAddress" text NULL;
+--
+-- Add field recipient to paymenthistoryrecord
+--
+ALTER TABLE "PaymentHistoryRecord" ADD COLUMN "recipient" jsonb NULL;
+--
+-- [CUSTOM] Revert Statement Timeout to default amount - 10 secs
+--
+SET statement_timeout = '10s';
+COMMIT;
+
+    `)
+}
+
+exports.down = async (knex) => {
+    await knex.raw(`
+    BEGIN;
+--
+-- Add field recipient to paymenthistoryrecord
+--
+ALTER TABLE "PaymentHistoryRecord" DROP COLUMN "recipient" CASCADE;
+--
+-- Add field rawAddress to paymenthistoryrecord
+--
+ALTER TABLE "PaymentHistoryRecord" DROP COLUMN "rawAddress" CASCADE;
+--
+-- Add field recipient to payment
+--
+ALTER TABLE "Payment" DROP COLUMN "recipient" CASCADE;
```

Comment:
why CASCADE everywhere?

Reply:
It's default. Dangerous if fields were relations, but they are not